spec/controllers/ingest_from_ftp_controller_spec.rb

```diff
@@ -150,4 +167,63 @@
       end
     end
   end
+
+  describe 'POST #delete_packages' do
+    context 'as an admin' do
+      before do
+        allow(controller).to receive(:authorize!).with(:read, :admin_dashboard).and_return(true)
+        allow(controller).to receive(:current_user).and_return(user)
+      end
+
+      it 'deletes a proquest ingest job and returns to the same page' do
+        selected_filenames = ['etdadmin_upload_3806.zip']
+        post :delete_packages, params: { source: 'proquest', selected_filenames: selected_filenames }, session: valid_session
+        expect(response).to redirect_to(ingest_from_ftp_path(source: 'proquest'))
+
+        package_filepaths = Dir[File.join(proquest_dir.to_s, '*.zip')]
+        expect(package_filepaths.length).to eq 1
+        expect(deleted_files_exist?(package_filepaths, selected_filenames)).to eq false
+      end
+
+      it 'deletes a sage ingest job and returns to the same page' do
+        selected_filenames = ['1177_01605976231158397.zip']
+        post :delete_packages, params: { source: 'sage', selected_filenames: selected_filenames }, session: valid_session
+        expect(response).to redirect_to(ingest_from_ftp_path(source: 'sage'))
+
+        package_filepaths = Dir[File.join(sage_dir.to_s, '*.zip')]
+        expect(package_filepaths.length).to eq 1
+        expect(deleted_files_exist?(package_filepaths, selected_filenames)).to eq false
+      end
+
+      it 'deletes an array of packages and returns to the same page' do
+        post :delete_packages, params: { source: 'proquest', selected_filenames: ['etdadmin_upload_3806.zip', 'etdadmin_upload_942402.zip']},
+          session: valid_session
+        expect(response).to redirect_to(ingest_from_ftp_path(source: 'proquest'))
+
+        package_filepaths = Dir[File.join(proquest_dir.to_s, '*.zip')]
+        expect(package_filepaths.empty?).to eq true
```

Comment:
```suggestion
        expect(package_filepaths).to be_empty
```
Just a handy built in check, same outcome of course